templates/firefox-ios.xliff

```diff
@@ -4110,6 +4136,10 @@ Private Tab</source>
         <source>Learn more about Firefox Suggest</source>
         <note>In the search page of the Settings menu, the title for the link to the SUMO Page about Firefox Suggest.</note>
       </trans-unit>
+      <trans-unit id="Settings.Search.Suggest.PrivateSession.Description.v125" xml:space="preserve">
+        <source>Show suggestions from %@ Suggest in private sessions</source>
```

Comment:
Since this is a feature, we should hardcode Firefox here (and remove the reference to the placeholder in the comment)

Reply:
https://github.com/mozilla-mobile/firefox-ios/pull/19174